activejob/lib/active_job/exceptions.rb

```diff
@@ -126,17 +126,20 @@ def retry_job(options = {})
     end
 
     private
-      def determine_delay(seconds_or_duration_or_algorithm:, executions:, jitter: nil)
-        jitter ||= self.class.retry_jitter
+      JITTER_DEFAULT = Object.new
+
+      def determine_delay(seconds_or_duration_or_algorithm:, executions:, jitter: JITTER_DEFAULT)
+        jitter = jitter == JITTER_DEFAULT ? self.class.retry_jitter : (jitter || 0.0)
+
         case seconds_or_duration_or_algorithm
         when :exponentially_longer
-          ((executions**4) + (Kernel.rand((executions**4) * jitter))) + 2
-        when ActiveSupport::Duration
-          duration = seconds_or_duration_or_algorithm.to_i
-          duration + Kernel.rand(duration * jitter)
-        when Integer
-          seconds = seconds_or_duration_or_algorithm
-          seconds + (Kernel.rand(seconds * jitter).ceil)
+          delay = executions**4
+          delay_jitter = determine_jitter_for_delay(delay, jitter)
+          delay + delay_jitter + 2
+        when ActiveSupport::Duration, Integer
+          delay = seconds_or_duration_or_algorithm.to_i
+          delay_jitter = determine_jitter_for_delay(delay, jitter)
+          delay + delay_jitter
```

Comment:
Two `when` cases were combined here. They were extremely similar before so I'm not sure if there's a style preference in Rails for keeping cases to a single condition. Happy to change this back if so.